#!/usr/bin/env python
PACKAGE = "potbot_filter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("threshold_point_num", int_t, 0, "A int parameter", 30, 0, 100)
gen.add("squre_width", double_t, 0, "A double parameter", 0.1, 0, 1)

exit(gen.generate(PACKAGE, "potbot_filter", "Clustering"))